Classes/Core/Functional/Framework/DataHandling/Scenario/EntityConfiguration.php

```diff
@@ -86,6 +90,11 @@ public function getParentColumnName(): ?string
         return $this->parentColumnName;
     }
 
+    public function getParentRelationColumnName(): ?string
+    {
+        return $this->parentRelationColumnName;
+    }
```

Comment:
Setting a parent relation column name does not make sense here as a table can be used in different tables and fields. So the relation is defined by the parent and not by the entity itself.